test/chat_api/users_test.exs

```diff
@@ -1,12 +1,45 @@
 defmodule ChatApi.UsersTest do
   use ChatApi.DataCase, async: true
+  require Logger
   @moduledoc false
 
   import ChatApi.Factory
 
+  alias ChatApi.Repo
   alias ChatApi.Users
   alias ChatApi.Users.{User, UserProfile, UserSettings}
 
+
+  describe "user" do
+    setup do
+      {:ok, user: insert(:user)}
```

Comment:
Where does the `insert` function come from?

Reply:
The insert function is coming from this https://github.com/papercups-io/papercups/blob/master/test/support/factory.ex 

In `user_test.exs` we have call `  import ChatApi.Factory`.  The `factory.ex` file calls uses [ExMachina.ecto](https://hexdocs.pm/ex_machina/ExMachina.Ecto.html) which gives you that `insert` function